core/templates/components/summary-tile/collection-summary-tile.component.html

```diff
@@ -58,4 +103,87 @@ <h3 class="activity-title protractor-test-collection-summary-tile-title">{{ getC
     position: absolute;
     width: 36px;
   }
+
+  .mobile-collection-card {
```

Comment:
A lot of the CSS I've added for the mobile versions of the exploration and collection cards is very similar, and could be moved to the global CSS file so that it is shared between both of the cards.

However, I've added the CSS separately in both the files because this way, we have more control over each individual card, and if any changes need to be made in the future, we can make changes for one card, without disturbing the other card.

I'd like to know a reviewer's opinion on which of the two approaches is better.

Reply:
Share the CSS which is common. The styles can be overridden from both the components individually when required so the control argument doesn't help.

Reply:
Done!